editor/editor_scale.cpp

```diff
@@ -33,7 +33,11 @@
 float EditorScale::_scale = 1.0f;
 
 void EditorScale::set_scale(float p_scale) {
-	_scale = p_scale;
+	if (p_scale > 0) {
```

Comment:
If scale is too large it will break the editor as well, so probably better to have it in some reasonable range, like
```c++
	_scale = CLAMP(p_scale, 0.1, 10);
```

Reply:
This won't work with `NaN` so it has to be clamped manually on the line below

Reply:
I think we can go with 0.5-3, since it's the float range of dragging the slider. This setting should allow for more accurate control in the reasonable range :)